syntax = "proto3";
package messages;

message HelloRequest {}
message HelloResponse {
    string message=1;
}

service HelloGrain {
	rpc SayHello(HelloRequest) returns (HelloResponse) {}
}